resolwe/flow/executors/local/run.py

```diff
@@ -61,12 +61,12 @@ async def terminate(self):
         try:
             self.proc.terminate()
         except ProcessLookupError:
-            # Process has already been terminated. Log exception and continue.
-            logger.exception("While terminating process with PID %s", self.proc.pid)
-
-        await asyncio.wait_for(self.proc.wait(), self.kill_delay)
-        if self.proc.returncode is None:
-            self.proc.kill()
-        await self.proc.wait()
+            # Process has already been terminated.
+            pass
+        else:
+            await asyncio.wait_for(self.proc.wait(), self.kill_delay)
```

Comment:
This line will raise asyncio.TimeoutError exception if timeout occured.
I suggest using suppress.

Reply:
Good point! Fixed.